src/Locations.ts

```diff
@@ -11,8 +11,16 @@ import type Space from './Space.js';
 import { ERR_NOT_ENTERED_SPACE } from './Errors.js';
 import SpaceUpdate from './SpaceUpdate.js';
 
+export namespace LocationsEvents {
```

Comment:
Would you mind explaining your objection to my use of namespaces in this case, given that I'm using them as a way to organise types?

I am also happy to change it, but I don't think I understand what you mean by using a module, would you mind showing me an example?

Reply:
hey @lawrence-forooghian thought about this and I think it's ok to use them _just for types_.

My objection for using namespaces is that they do not exist in JavaScript and so if you add code into them they will have to compile into _something else_. They were deprecated at some point in TS but I think that decision was reversed (because of how many people use them maybe). Apart from them not existing in JavaScript, the main issue is they duplicate what modules do (ie. organise code, namespace methods and properties).